views/partials/replyForm.hbs

```diff
@@ -0,0 +1,5 @@
+<form id="{{id}}-reply" method="POST" action="/messages/{{id}}/reply" style="display: none;">
```

Comment:
Give these a class like `class="reply-form"`. Then have a standalone JS file included in the browser that does the following:

```javascript
function toggleElement(el) {
  el.style.display = el.style.display === 'none ? 'initial' : 'none';
}
```

document.addEventListener('DOMContentLoaded', function() {
  document.querySelectorAll('form.reply-form').forEach(function(form) {
    form.querySelector('button.reply').addEventListener('click', () => toggleElement(form));
  });
});
```

You keep the HTML clean and your JS all lives in one place. You only have to annotate your HTML with classes and other attributes so that your JavaScript knows how to get a handle on the appropriate parts of your webpage.